[Doc][azsecrets] outdated code sample for Update Secret metadata

- What happened?
copy and paste code snippet from Update Secret metadata sample but it doesn't build



- What did you expect or want to happen?
correct code:
```diff
	properties := azsecrets.Properties{
		ContentType: &contentType,
		SecretAttributes: &azsecrets.Attributes{
			Enabled: to.BoolPtr(false),
		},
	}
	updateResp, err := client.UpdateSecretProperties(context.Background(), "secretName", properties, nil)
```


logging sample is also incorrect
![image](https://user-images.githubusercontent.com/7583839/145115722-8e009516-1345-472c-9ccc-2e2a52004d16.png)

Now I am hitting a nil point deref
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x8 pc=0x2c818f]

goroutine 1 [running]:
github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets.Attributes.toGenerated(...)
        C:/Users/yumeng/go/pkg/mod/github.com/!azure/azure-sdk-for-go/sdk/keyvault/azsecrets@v0.3.0/models.go:93
github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets.Properties.toGenerated({0xc000105200, 0xc00008a840, 0x0})
        C:/Users/yumeng/go/pkg/mod/github.com/!azure/azure-sdk-for-go/sdk/keyvault/azsecrets@v0.3.0/client.go:442 +0x8f
github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets.(*Client).UpdateSecretProperties(0xc000118150, {0x3b57f0, 0xc00011a010}, {0x344857, 0xa}, {0xc000105200, 0xc00008a840, 0x0}, 0x3292a0)
        C:/Users/yumeng/go/pkg/mod/github.com/!azure/azure-sdk-for-go/sdk/keyvault/azsecrets@v0.3.0/client.go:459 +0xd5
main.main()
        C:/working/go/kv-samples/src/main.go:48 +0x74c
```

https://github.com/Azure/azure-sdk-for-go/blob/88d611e9ded5369c62e0776a641d1e019ad22ecc/sdk/keyvault/azsecrets/models.go#L93

--- a/sdk/keyvault/azsecrets/README.md
+++ b/sdk/keyvault/azsecrets/README.md
@@ -237,12 +237,12 @@ To obtain more detailed logging, including request/response bodies and header va
 ```go
 import azlog "github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
 // Set log to output to the console
-log.SetListener(func(cls log.Event, msg string) {
+azlog.SetListener(func(cls azlog.Event, msg string) {
     fmt.Println(msg)
 })
 
 // Includes only requests and responses in credential logs
-log.SetEvents(log.EventRequest, log.EventResponse)
+azlog.SetEvents(azlog.EventRequest, azlog.EventResponse)
 ```
 
 > CAUTION: logs from credentials contain sensitive information.

--- a/sdk/keyvault/azsecrets/README.md
+++ b/sdk/keyvault/azsecrets/README.md
@@ -6,11 +6,11 @@ Azure Key Vault helps securely store and control access to tokens, passwords, ce
 ## Getting started
 
 ### Install packages
-Install `azsecrets` and [azure-identity][azidentity_goget]:
+Install `azsecrets` and [azidentity][azidentity_goget]:
 ```
 go get -u github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets
 ```
-[azure-identity][azure_identity] is used for Azure Active Directory authentication as demonstrated below.
+[azidentity][azure_identity] is used for Azure Active Directory authentication as demonstrated below.
 ```
 go get -u github.com/Azure/azure-sdk-for-go/sdk/azidentity
 ```
@@ -59,7 +59,7 @@ go get -u github.com/Azure/azure-sdk-for-go/sdk/azidentity
   > The `"vaultUri"` property is the `vaultUrl` used by [azsecrets.NewClient][secret_client_docs]
 
 ### Authenticate the client
-This document demonstrates using [DefaultAzureCredential][default_cred_ref] to authenticate as a service principal. However, [Client][secret_client_docs] accepts any [azure-identity][azure_identity] credential. See the [azure-identity][azure_identity] documentation for more information about other credentials.
+This document demonstrates using [DefaultAzureCredential][default_cred_ref] to authenticate as a service principal. However, [Client][secret_client_docs] accepts any [azidentity][azure_identity] credential. See the [azidentity][azure_identity] documentation for more information about other credentials.
 
 
 #### Create a service principal (optional)

--- a/sdk/keyvault/azsecrets/README.md
+++ b/sdk/keyvault/azsecrets/README.md
@@ -164,16 +164,17 @@ contentType := "text/plain"
 
 // We will also disable the secret for further use
 
-properties := azsecrets.SecretProperties{
-    ContentType: &contentType,
-    SecretAttributes: &azsecrets.SecretAttributes{
-        Attributes: &azsecrets.Attributes{
-            Enabled: to.BoolPtr(false),
-        },
+properties := Properties{
+    ContentType: to.StringPtr("password"),
+    Tags: map[string]string{
+        "Tag1": "TagVal1",
+    },
+    SecretAttributes: &Attributes{
+        Enabled: to.BoolPtr(true),
     },
 }
 
-resp, err := client.UpdateSecretProperties(context.Background(), "mySecretName", &properties, nil)
+resp, err := client.UpdateSecretProperties(context.Background(), "mySecretName", properties, nil)
 if err != nil {
     // handle error...
 }
@@ -236,12 +237,12 @@ To obtain more detailed logging, including request/response bodies and header va
 ```go
 import azlog "github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
 // Set log to output to the console
-log.SetListener(func(cls log.Classification, msg string) {
-		fmt.Println(msg) // printing log out to the console
+log.SetListener(func(cls log.Event, msg string) {
+    fmt.Println(msg)
 })
 
 // Includes only requests and responses in credential logs
-log.SetClassifications(log.Request, log.Response)
+log.SetEvents(log.EventRequest, log.EventResponse)
 ```
 
 > CAUTION: logs from credentials contain sensitive information.

--- a/sdk/keyvault/azsecrets/README.md
+++ b/sdk/keyvault/azsecrets/README.md
@@ -164,12 +164,12 @@ contentType := "text/plain"
 
 // We will also disable the secret for further use
 
-properties := Properties{
+properties := azsecrets.Properties{
     ContentType: to.StringPtr("password"),
     Tags: map[string]string{
         "Tag1": "TagVal1",
     },
-    SecretAttributes: &Attributes{
+    SecretAttributes: &azsecrets.Attributes{
         Enabled: to.BoolPtr(true),
     },
 }

--- a/sdk/keyvault/azsecrets/client_test.go
+++ b/sdk/keyvault/azsecrets/client_test.go
@@ -9,6 +9,7 @@ package azsecrets
 import (
 	"context"
 	"errors"
+	"fmt"
 	"net/http"
 	"os"
 	"strings"
@@ -16,6 +17,7 @@ import (
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
@@ -270,7 +272,7 @@ func TestUpdateSecretProperties(t *testing.T) {
 	client, err := createClient(t)
 	require.NoError(t, err)
 
-	secret, err := createRandomName(t, "secret")
+	secret, err := createRandomName(t, "secret2")
 	require.NoError(t, err)
 	value, err := createRandomName(t, "value")
 	require.NoError(t, err)
@@ -288,6 +290,9 @@ func TestUpdateSecretProperties(t *testing.T) {
 		ContentType: to.StringPtr("password"),
 		Tags: map[string]string{
 			"Tag1": "TagVal1",
+		},
+		SecretAttributes: &Attributes{
+			Enabled: to.BoolPtr(true),
 		},
 	}
 
@@ -407,6 +412,31 @@ func TestTimeout(t *testing.T) {
 	c := context.Background()
 	c, cancelFunc := context.WithTimeout(c, 10*time.Second)
 	defer cancelFunc()
+
+	start := time.Now()
+	_, err = client.GetSecret(c, "nonexistentsecret", nil)
+	require.Error(t, err)
+	require.ErrorIs(t, err, context.DeadlineExceeded)
+	require.Less(t, time.Since(start).Seconds(), 11.0)
+	require.Greater(t, time.Since(start).Seconds(), 9.0)
+}
+
+func TestLogging(t *testing.T) {
+	fakeKVUrl := "https://test-sync-time-dummy.vault.azure.net/"
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	require.NoError(t, err)
+
+	client, err := NewClient(fakeKVUrl, cred, nil)
+	require.NoError(t, err)
+
+	c := context.Background()
+	c, cancelFunc := context.WithTimeout(c, 10*time.Second)
+	defer cancelFunc()
+
+	log.SetListener(func(cls log.Event, msg string) {
+		fmt.Println(msg)
+	})
+	log.SetEvents(log.EventRequest, log.EventResponse)
 
 	start := time.Now()
 	_, err = client.GetSecret(c, "nonexistentsecret", nil)

--- a/sdk/keyvault/azsecrets/consts.go
+++ b/sdk/keyvault/azsecrets/consts.go
@@ -66,21 +66,31 @@ func deletionRecoveryLevelFromGenerated(i internal.DeletionRecoveryLevel) Deleti
 	}
 }
 
-func (d DeletionRecoveryLevel) toGenerated() internal.DeletionRecoveryLevel {
-	if d == CustomizedRecoverable {
-		return internal.DeletionRecoveryLevelCustomizedRecoverable
-	} else if d == CustomizedRecoverableProtectedSubscription {
-		return internal.DeletionRecoveryLevelCustomizedRecoverableProtectedSubscription
-	} else if d == CustomizedRecoverablePurgeable {
-		return internal.DeletionRecoveryLevelCustomizedRecoverablePurgeable
-	} else if d == Purgeable {
-		return internal.DeletionRecoveryLevelPurgeable
-	} else if d == Recoverable {
-		return internal.DeletionRecoveryLevelRecoverable
-	} else if d == RecoverableProtectedSubscription {
-		return internal.DeletionRecoveryLevelRecoverableProtectedSubscription
+func (d *DeletionRecoveryLevel) toGenerated() *internal.DeletionRecoveryLevel {
+	if d == nil {
+		return nil
+	}
+	if *d == CustomizedRecoverable {
+		i := internal.DeletionRecoveryLevelCustomizedRecoverable
+		return i.ToPtr()
+	} else if *d == CustomizedRecoverableProtectedSubscription {
+		i := internal.DeletionRecoveryLevelCustomizedRecoverableProtectedSubscription
+		return i.ToPtr()
+	} else if *d == CustomizedRecoverablePurgeable {
+		i := internal.DeletionRecoveryLevelCustomizedRecoverablePurgeable
+		return i.ToPtr()
+	} else if *d == Purgeable {
+		i := internal.DeletionRecoveryLevelPurgeable
+		return i.ToPtr()
+	} else if *d == Recoverable {
+		i := internal.DeletionRecoveryLevelRecoverable
+		return i.ToPtr()
+	} else if *d == RecoverableProtectedSubscription {
+		i := internal.DeletionRecoveryLevelRecoverableProtectedSubscription
+		return i.ToPtr()
 	} else {
-		return internal.DeletionRecoveryLevelRecoverablePurgeable
+		i := internal.DeletionRecoveryLevelRecoverablePurgeable
+		return i.ToPtr()
 	}
 }
 

--- a/sdk/keyvault/azsecrets/models.go
+++ b/sdk/keyvault/azsecrets/models.go
@@ -88,9 +88,13 @@ type Attributes struct {
 }
 
 func (s Attributes) toGenerated() *internal.SecretAttributes {
+	var recLevel *internal.DeletionRecoveryLevel
+	if s.RecoveryLevel != nil {
+		recLevel = s.RecoveryLevel.toGenerated().ToPtr()
+	}
 	return &internal.SecretAttributes{
 		RecoverableDays: s.RecoverableDays,
-		RecoveryLevel:   s.RecoveryLevel.toGenerated().ToPtr(),
+		RecoveryLevel:   recLevel,
 		Attributes: internal.Attributes{
 			Enabled:   s.Enabled,
 			Expires:   s.Expires,

--- a/sdk/keyvault/azsecrets/models.go
+++ b/sdk/keyvault/azsecrets/models.go
@@ -88,13 +88,9 @@ type Attributes struct {
 }
 
 func (s Attributes) toGenerated() *internal.SecretAttributes {
-	var recLevel *internal.DeletionRecoveryLevel
-	if s.RecoveryLevel != nil {
-		recLevel = s.RecoveryLevel.toGenerated().ToPtr()
-	}
 	return &internal.SecretAttributes{
 		RecoverableDays: s.RecoverableDays,
-		RecoveryLevel:   recLevel,
+		RecoveryLevel:   s.RecoveryLevel.toGenerated(),
 		Attributes: internal.Attributes{
 			Enabled:   s.Enabled,
 			Expires:   s.Expires,

--- a/sdk/keyvault/azsecrets/testdata/recordings/TestUpdateSecretProperties.json
+++ b/sdk/keyvault/azsecrets/testdata/recordings/TestUpdateSecretProperties.json
@@ -1,18 +1,18 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret22319658013?api-version=7.2",
       "RequestMethod": "PUT",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "PUT",
-        ":path": "/secrets/secret2319658013?api-version=7.2",
+        ":path": "/secrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Content-Length": "0",
         "Content-Type": "application/json",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 401,
@@ -20,7 +20,7 @@
         "Cache-Control": "no-cache",
         "Content-Length": "97",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:32 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:44 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
@@ -28,8 +28,8 @@
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "7cf2de70-a51a-4dc5-8da4-b40556513244",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "c9bdc9c9-e973-4288-9a7d-9c1f5639d6a4",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
@@ -40,19 +40,19 @@
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret22319658013?api-version=7.2",
       "RequestMethod": "PUT",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "PUT",
-        ":path": "/secrets/secret2319658013?api-version=7.2",
+        ":path": "/secrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
         "Content-Length": "53",
         "Content-Type": "application/json",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": {
         "attributes": {},
@@ -62,26 +62,26 @@
       "StatusCode": 200,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "276",
+        "Content-Length": "277",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:33 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:45 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "f5852716-91cb-43f1-8353-c52718d4a44b",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "bf4d2070-797d-4a65-855d-c71841136e7d",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
         "value": "value2319658013",
-        "id": "https://rosebud.vault.azure.net/secrets/secret2319658013/fb7339dffdbc4af6a8a9b74a090bb324",
+        "id": "https://rosebud.vault.azure.net/secrets/secret22319658013/486a573e975c42c883085d35c077ae48",
         "attributes": {
           "enabled": true,
-          "created": 1636487073,
-          "updated": 1636487073,
+          "created": 1638993885,
+          "updated": 1638993885,
           "recoveryLevel": "CustomizedRecoverable\u002BPurgeable",
           "recoverableDays": 7
         },
@@ -89,42 +89,42 @@
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret2319658013/?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret22319658013/?api-version=7.2",
       "RequestMethod": "GET",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "GET",
-        ":path": "/secrets/secret2319658013/?api-version=7.2",
+        ":path": "/secrets/secret22319658013/?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 200,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "276",
+        "Content-Length": "277",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:33 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:45 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "80d5f1d9-93e4-4e4d-9899-8bd1926f8176",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "a2df6b4d-4bc1-493b-ba62-9b6c5f47eb55",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
         "value": "value2319658013",
-        "id": "https://rosebud.vault.azure.net/secrets/secret2319658013/fb7339dffdbc4af6a8a9b74a090bb324",
+        "id": "https://rosebud.vault.azure.net/secrets/secret22319658013/486a573e975c42c883085d35c077ae48",
         "attributes": {
           "enabled": true,
-          "created": 1636487073,
-          "updated": 1636487073,
+          "created": 1638993885,
+          "updated": 1638993885,
           "recoveryLevel": "CustomizedRecoverable\u002BPurgeable",
           "recoverableDays": 7
         },
@@ -132,21 +132,24 @@
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret2319658013/?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret22319658013/?api-version=7.2",
       "RequestMethod": "PATCH",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "PATCH",
-        ":path": "/secrets/secret2319658013/?api-version=7.2",
+        ":path": "/secrets/secret22319658013/?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "Content-Length": "52",
+        "Content-Length": "82",
         "Content-Type": "application/json",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": {
+        "attributes": {
+          "enabled": true
+        },
         "contentType": "password",
         "tags": {
           "Tag1": "TagVal1"
@@ -155,26 +158,26 @@
       "StatusCode": 200,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "291",
+        "Content-Length": "292",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:33 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:45 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "032b7eae-f22b-4a44-a7ee-0c9829d5c291",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "ea01d377-712f-4819-8a7d-aac88dc9c01e",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
         "contentType": "password",
-        "id": "https://rosebud.vault.azure.net/secrets/secret2319658013/fb7339dffdbc4af6a8a9b74a090bb324",
+        "id": "https://rosebud.vault.azure.net/secrets/secret22319658013/486a573e975c42c883085d35c077ae48",
         "attributes": {
           "enabled": true,
-          "created": 1636487073,
-          "updated": 1636487073,
+          "created": 1638993885,
+          "updated": 1638993886,
           "recoveryLevel": "CustomizedRecoverable\u002BPurgeable",
           "recoverableDays": 7
         },
@@ -184,43 +187,43 @@
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret2319658013/?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret22319658013/?api-version=7.2",
       "RequestMethod": "GET",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "GET",
-        ":path": "/secrets/secret2319658013/?api-version=7.2",
+        ":path": "/secrets/secret22319658013/?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 200,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "317",
+        "Content-Length": "318",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:33 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:45 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "1ca8cd63-6cc4-4328-a453-ff25c9ceaf88",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "3c4c774e-8d3c-4971-8b17-a62375ed6266",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
         "value": "value2319658013",
         "contentType": "password",
-        "id": "https://rosebud.vault.azure.net/secrets/secret2319658013/fb7339dffdbc4af6a8a9b74a090bb324",
+        "id": "https://rosebud.vault.azure.net/secrets/secret22319658013/486a573e975c42c883085d35c077ae48",
         "attributes": {
           "enabled": true,
-          "created": 1636487073,
-          "updated": 1636487073,
+          "created": 1638993885,
+          "updated": 1638993886,
           "recoveryLevel": "CustomizedRecoverable\u002BPurgeable",
           "recoverableDays": 7
         },
@@ -230,45 +233,45 @@
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/secret22319658013?api-version=7.2",
       "RequestMethod": "DELETE",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "DELETE",
-        ":path": "/secrets/secret2319658013?api-version=7.2",
+        ":path": "/secrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 200,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "427",
+        "Content-Length": "429",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:33 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:46 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "a0eefbb5-9d2f-4829-a4ef-9e7fe7ec8aa5",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "6ca730e0-cbe3-4623-9dfd-d6cbb9df6bae",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
-        "recoveryId": "https://rosebud.vault.azure.net/deletedsecrets/secret2319658013",
-        "deletedDate": 1636487074,
-        "scheduledPurgeDate": 1637091874,
+        "recoveryId": "https://rosebud.vault.azure.net/deletedsecrets/secret22319658013",
+        "deletedDate": 1638993886,
+        "scheduledPurgeDate": 1639598686,
         "contentType": "password",
-        "id": "https://rosebud.vault.azure.net/secrets/secret2319658013/fb7339dffdbc4af6a8a9b74a090bb324",
+        "id": "https://rosebud.vault.azure.net/secrets/secret22319658013/486a573e975c42c883085d35c077ae48",
         "attributes": {
           "enabled": true,
-          "created": 1636487073,
-          "updated": 1636487073,
+          "created": 1638993885,
+          "updated": 1638993886,
           "recoveryLevel": "CustomizedRecoverable\u002BPurgeable",
           "recoverableDays": 7
         },
@@ -278,267 +281,156 @@
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret22319658013?api-version=7.2",
       "RequestMethod": "GET",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "GET",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
+        ":path": "/deletedsecrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 404,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "90",
+        "Content-Length": "91",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:33 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:46 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "f5f941ef-53b5-423a-bc98-ec0d07b26ccd",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "f8a99040-31a5-4c9d-8da4-33848dc9c759",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
         "error": {
           "code": "SecretNotFound",
-          "message": "Deleted Secret not found: secret2319658013"
+          "message": "Deleted Secret not found: secret22319658013"
         }
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret22319658013?api-version=7.2",
       "RequestMethod": "GET",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "GET",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
+        ":path": "/deletedsecrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 404,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "90",
+        "Content-Length": "91",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:33 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:46 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "a382c95a-0e30-4b4c-b893-3ee37b366ce6",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "d42b2f0a-0383-49ae-8838-09672b26b6aa",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
         "error": {
           "code": "SecretNotFound",
-          "message": "Deleted Secret not found: secret2319658013"
+          "message": "Deleted Secret not found: secret22319658013"
         }
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret22319658013?api-version=7.2",
       "RequestMethod": "GET",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "GET",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
+        ":path": "/deletedsecrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 404,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "90",
+        "Content-Length": "91",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:34 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:46 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "d2113160-f183-4ddf-b122-4665698c852d",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "bba5982f-2181-4f20-a39a-12039f5262ca",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
         "error": {
           "code": "SecretNotFound",
-          "message": "Deleted Secret not found: secret2319658013"
+          "message": "Deleted Secret not found: secret22319658013"
         }
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret22319658013?api-version=7.2",
       "RequestMethod": "GET",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "GET",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
+        ":path": "/deletedsecrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
-      },
-      "RequestBody": null,
-      "StatusCode": 404,
-      "ResponseHeaders": {
-        "Cache-Control": "no-cache",
-        "Content-Length": "90",
-        "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:34 GMT",
-        "Expires": "-1",
-        "Pragma": "no-cache",
-        "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
-        "X-Content-Type-Options": "nosniff",
-        "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
-        "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "9ecf9c8c-ae47-4365-ae31-f6689e7fc36b",
-        "X-Powered-By": "ASP.NET"
-      },
-      "ResponseBody": {
-        "error": {
-          "code": "SecretNotFound",
-          "message": "Deleted Secret not found: secret2319658013"
-        }
-      }
-    },
-    {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
-      "RequestMethod": "GET",
-      "RequestHeaders": {
-        ":authority": "localhost:5001",
-        ":method": "GET",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
-        ":scheme": "https",
-        "Accept": "application/json",
-        "Accept-Encoding": "gzip",
-        "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
-      },
-      "RequestBody": null,
-      "StatusCode": 404,
-      "ResponseHeaders": {
-        "Cache-Control": "no-cache",
-        "Content-Length": "90",
-        "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:34 GMT",
-        "Expires": "-1",
-        "Pragma": "no-cache",
-        "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
-        "X-Content-Type-Options": "nosniff",
-        "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
-        "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "a49ed116-67cb-4813-9058-788772d7546a",
-        "X-Powered-By": "ASP.NET"
-      },
-      "ResponseBody": {
-        "error": {
-          "code": "SecretNotFound",
-          "message": "Deleted Secret not found: secret2319658013"
-        }
-      }
-    },
-    {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
-      "RequestMethod": "GET",
-      "RequestHeaders": {
-        ":authority": "localhost:5001",
-        ":method": "GET",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
-        ":scheme": "https",
-        "Accept": "application/json",
-        "Accept-Encoding": "gzip",
-        "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
-      },
-      "RequestBody": null,
-      "StatusCode": 404,
-      "ResponseHeaders": {
-        "Cache-Control": "no-cache",
-        "Content-Length": "90",
-        "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:35 GMT",
-        "Expires": "-1",
-        "Pragma": "no-cache",
-        "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
-        "X-Content-Type-Options": "nosniff",
-        "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
-        "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "2945bf8b-ab4b-49a3-9b03-d8c81c6c3758",
-        "X-Powered-By": "ASP.NET"
-      },
-      "ResponseBody": {
-        "error": {
-          "code": "SecretNotFound",
-          "message": "Deleted Secret not found: secret2319658013"
-        }
-      }
-    },
-    {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
-      "RequestMethod": "GET",
-      "RequestHeaders": {
-        ":authority": "localhost:5001",
-        ":method": "GET",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
-        ":scheme": "https",
-        "Accept": "application/json",
-        "Accept-Encoding": "gzip",
-        "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 200,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Content-Length": "427",
+        "Content-Length": "429",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 09 Nov 2021 19:44:35 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:46 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "35bacc43-affd-4c09-99cf-c97a92b6799c",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "629e57fb-2ec6-47fd-8ae4-6db79e977a43",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": {
-        "recoveryId": "https://rosebud.vault.azure.net/deletedsecrets/secret2319658013",
-        "deletedDate": 1636487074,
-        "scheduledPurgeDate": 1637091874,
+        "recoveryId": "https://rosebud.vault.azure.net/deletedsecrets/secret22319658013",
+        "deletedDate": 1638993886,
+        "scheduledPurgeDate": 1639598686,
         "contentType": "password",
-        "id": "https://rosebud.vault.azure.net/secrets/secret2319658013/fb7339dffdbc4af6a8a9b74a090bb324",
+        "id": "https://rosebud.vault.azure.net/secrets/secret22319658013/486a573e975c42c883085d35c077ae48",
         "attributes": {
           "enabled": true,
-          "created": 1636487073,
-          "updated": 1636487073,
+          "created": 1638993885,
+          "updated": 1638993886,
           "recoveryLevel": "CustomizedRecoverable\u002BPurgeable",
           "recoverableDays": 7
         },
@@ -548,31 +440,31 @@
       }
     },
     {
-      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret2319658013?api-version=7.2",
+      "RequestUri": "https://fakekvurl.vault.azure.net/deletedsecrets/secret22319658013?api-version=7.2",
       "RequestMethod": "DELETE",
       "RequestHeaders": {
         ":authority": "localhost:5001",
         ":method": "DELETE",
-        ":path": "/deletedsecrets/secret2319658013?api-version=7.2",
+        ":path": "/deletedsecrets/secret22319658013?api-version=7.2",
         ":scheme": "https",
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "Authorization": "Sanitized",
-        "User-Agent": "azsdk-go-internal/v0.2.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
+        "User-Agent": "azsdk-go-internal/v0.3.1 azsdk-go-azcore/v0.20.0 (go1.17; Windows_NT)"
       },
       "RequestBody": null,
       "StatusCode": 204,
       "ResponseHeaders": {
         "Cache-Control": "no-cache",
-        "Date": "Tue, 09 Nov 2021 19:44:35 GMT",
+        "Date": "Wed, 08 Dec 2021 20:04:47 GMT",
         "Expires": "-1",
         "Pragma": "no-cache",
         "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
         "X-Content-Type-Options": "nosniff",
         "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=72.49.29.93;act_addr_fam=InterNetwork;",
         "x-ms-keyvault-region": "westus2",
-        "x-ms-keyvault-service-version": "1.9.150.1",
-        "x-ms-request-id": "62f05d02-18bb-4938-adff-b64927ae7d3a",
+        "x-ms-keyvault-service-version": "1.9.195.1",
+        "x-ms-request-id": "af0005e6-bda1-4588-a01b-fde361b6c23c",
         "X-Powered-By": "ASP.NET"
       },
       "ResponseBody": null

